src/components/BookCard/BookCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styles from './BookCard.module.css'
+import favorite from '../../assets/favorite.svg'
+import book from '../../assets/bookExample.svg'
+
+const BookCard = () => {
+  return (
+    <div className={styles.bookCard}>
```

Comment:
Set the same width as in Figma for the Card component from the Catalog, the cards from the Popular section are wider